COLLABORATOR_GUIDE.md

```diff
@@ -255,14 +289,32 @@ When you send your pull request, consider including information about
 whether your change is breaking. If you think your patch can be backported,
 please feel free to include that information in the PR thread.
 
-#### Who is doing the backporting?
-
-The current plan is for commits to cherry pick into a staging branch (e.g. v4.x-staging),
-which can be done by anyone. The preference would be for the individual landing the commit
-on master to backport to staging branches if it is appropriate.
+Several LTS related issue and PR labels have been provided:
+
+* `lts-watch-v4.x` - tells the LTS WG that the issue/PR needs to be considered
+  for landing in the `v4.x-staging` branch.
+* `lts-watch-v0.10` - tells the LTS WG that the issue/PR needs to be considered
+  for landing in the `v0.10-staging` branch.
+* `lts-watch-v0.12` - tells the LTS WG that the issue/PR needs to be considered
+  for landing in the `v0.12-staging` branch.
+* `land-on-v4.x` - tells the release team that the commit should be landed
+  in a future v4.x release
+* `land-on-v0.10` - tells the release team that the commit should be landed
+  in a future v0.10 release
+* `land-on-v0.12` - tells the release team that the commit should be landed
+  in a future v0.12 release
+  
+Any collaborator can attach these labels to any PR/issue. As commits are
```

Comment:
I created new labels `v0.10.x`, `v0.12.x`, `v4.x`, `v5.x` and `master` to identify issues (only issues) that are _confirmed_ on these respective versions. Are some of them duplicates of `lts-watch-v4.x`? I wouldn't think so, since `lts-watch-v4.x` seems to mean _"consider landing this"_ instead of _"this issue can be reproduced on this releases line"_, but I would like to avoid the use of too many labels that overlap each other.